pkg/network/node/vxlan_monitor_test.go

```diff
@@ -1,315 +0,0 @@
-// +build linux
-
-package node
-
-import (
-	"testing"
-
-	"github.com/openshift/sdn/pkg/network/node/ovs"
-)
-
-func packetsIn(ovsif ovs.Interface, counts map[string]int, nodeIP string) {
-	counts[nodeIP] += 1
-	otx := ovsif.NewTransaction()
-	otx.DeleteFlows("table=10, tun_src=%s", nodeIP)
-	otx.AddFlow("table=10, n_packets=%d, tun_src=%s, actions=goto_table:30", counts[nodeIP], nodeIP)
-	err := otx.Commit()
-	if err != nil {
-		panic("can't happen: " + err.Error())
-	}
-}
-
-func packetsOut(ovsif ovs.Interface, counts map[uint32]int, nodeIP string, vnid uint32) {
-	counts[vnid] += 1
-	otx := ovsif.NewTransaction()
-	otx.DeleteFlows("table=100, dummy=%s, reg0=%d", nodeIP, vnid)
-	otx.AddFlow("table=100, n_packets=%d, dummy=%s, reg0=%d, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", counts[vnid], nodeIP, vnid, nodeIP)
-	err := otx.Commit()
-	if err != nil {
-		panic("can't happen: " + err.Error())
-	}
-}
-
-func peekUpdate(updates chan struct{}, evm *egressVXLANMonitor) []egressVXLANNode {
-	select {
-	case <-updates:
-		return evm.GetUpdates()
-	default:
-		return nil
-	}
-}
-
-func TestEgressVXLANMonitor(t *testing.T) {
```

Comment:
seems like probably we need more testing in `egressip_test.go` to make up for the lost testing in `vxlan_monitor_test.go`

Reply:
I'll try to write something up. Although, `vxlan_monitor_test.go` only tested the logic that founded itself on parsing OVS stats. There were no tests for what concerns polling nodes and mocking connectivity issues.

Reply:
ok, if we haven't lost any non-ovs-parsing testing then we don't need anything more